openwisp_radius/migrations/0025_sms_verification.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.1.13 on 2021-11-12 15:01
```

Comment:
Why there are two migrations files? Can you merge them into one?

Reply:
Will do.